pom.xml

```diff
@@ -554,7 +554,7 @@
           <artifactId>maven-compiler-plugin</artifactId>
           <version>3.10.1</version>
           <configuration>
-            <release>11</release>
+            <release>8</release>
```

Comment:
Since this will downgrade the Java version in our repo, I'm curious if it will cause any dependency conflict to other customers that are using Java 11? I saw when Komal upgraded this repo to Java 11, she also upgraded a couple of [dependencies here](https://github.com/apache/helix/commit/a180628893d148da32385f1baef88f60c1ffceb2). Do we want to revert them back as well?